daemon/src/control.rs

```diff
@@ -0,0 +1,316 @@
+//! Utility for fixture data in the monorepo.
```

Comment:
Not sure this module should make it over as the fixtures and git-platinum handling seems to be upstream specific, unless I'm missing something.